doc/source/building/index.rst

```diff
@@ -376,6 +376,37 @@ interface is self-documenting, so please see ``python dev.py --help`` and
 ``python dev.py <subcommand> --help`` for detailed guidance.
 
 
+.. admonition:: IDE support & editable installs
+
+    While the ``dev.py`` interface is our recommended way of working on SciPy,
+    it has one limitation: because of the custom install location, setting
+    breakpoints inside an IDE or debugger does not work as expected. This will
+    only work with an *in-place build* (or "editable install").
+
+    Editable installs are supported. It is important to understand that **you
+    must use either an editable install or dev.py, but not both**. If you use
```

Comment:
```suggestion
    may use either an editable install or dev.py, but not both**. If you use
```

To make the rule easier to remember, maybe this can help the reader build a mental model of why.

The real limitation is that both can't be done within a single clone of the repository, right? It could be as simple as adding the qualification "in a given repository clone", but a separate sentence to explain would also work.

Reply:
sure, I'll add that phrase. I don't really want to explain everything in detail, that will start duplication the meson-python docs.